File: Preload/MyPreLoadData/java/app/src/main/java/com/dicoding/picodiploma/mypreloaddata/database/DatabaseContract.java
package com.dicoding.picodiploma.mypreloaddata.database;

import android.provider.BaseColumns;

/**
 * Created by dicoding on 10/18/2017.
 */

class DatabaseContract {

    public static final String TABLE_NAME = "table_mahasiswa";

    public static final class MahasiswaColumns implements BaseColumns {

        // Mahasiswa nama
        public static final String NAMA = "nama";
        // Mahasiswa nim
        public static final String NIM = "nim";

    }
}

File: Preload/MyPreLoadData/kotlin/app/src/main/java/com/dicoding/picodiploma/mypreloaddata/database/DatabaseContract.kt
package com.dicoding.picodiploma.mypreloaddata.database

import android.provider.BaseColumns

/**
 * Created by dicoding on 10/18/2017.
 */

internal object DatabaseContract {

    var TABLE_NAME = "table_mahasiswa"

    internal class MahasiswaColumns : BaseColumns {
        companion object {
            const val NAMA = "nama"
            const val NIM = "nim"
        }
    }
}
